.github/workflows/deploy.yml

```diff
@@ -230,7 +230,10 @@ jobs:
   finishing-prep:
     name: Finishing Prep
     needs: [deploy-infra, begin-deployment, deploy-app]
-    if: always() && needs.deploy-infra.result == 'success' && needs.deploy-app.result == 'success'
+    if: |
+      always() 
+      && (needs.deploy-infra.result == 'success' || needs.deploy-infra.result == 'skipped') 
+      && (needs.deploy-app.result == 'success' || needs.deploy-app.result == 'skipped')
```

Comment:
I was seeing that Cypress would get skipped, because this step would sometimes get skipped, and a skip is not considered a success in the same way. Anyway, this makes it so we always run finishing prep and then get a Cypress run.